bluetooth/src/commonMain/kotlin/scanner/Scanner.kt

```diff
@@ -71,21 +76,29 @@ interface Scanner {
 }
 
 abstract class BaseScanner constructor(
-    settings: Settings,
+    private val settings: Settings,
     private val coroutineScope: CoroutineScope
 ) : Scanner, CoroutineScope by coroutineScope {
 
     companion object {
+        private const val LOG_TAG = "Bluetooth Scanner"
         const val DEFAULT_EVENT_BUFFER_SIZE = 256
     }
 
     data class Settings(
         val permissions: Permissions,
         val autoRequestPermission: Boolean = true,
         val autoEnableSensors: Boolean = true,
-        val eventBufferSize: Int = DEFAULT_EVENT_BUFFER_SIZE
+        val eventBufferSize: Int = DEFAULT_EVENT_BUFFER_SIZE,
+        val logLevel: LogLevel = LogLevel.NONE
     )
 
+    enum class LogLevel {
```

Comment:
Can we reuse already existing log level (from logger for example)? It's definitely duplication